### ISSUE TITLE
CSV-Import broken after updating Humhub to 1.13.0

### ISSUE THREAD

**Issue by weyhmueller** [humhub/massuserimport-issues#9]
It seems that https://github.com/humhub/humhub/pull/5872 is breaking CSV import with the massuserimport module.

We use PHP 8.1. When importing, every line in the CSV file causes the following error:

```
Ein allgemeiner Fehler ist aufgetreten. Indirect modification of overloaded property
humhub\modules\massuserimport\models\MassuserimportPassword::$user has no effect
```
After setting `error_reporting` to exclude `E_NOTICE` a fatal error is thrown instead:

```
Error: Attempt to assign property "auth_key" on null in /var/www/virtual/customer/html/protected/humhub/modules/user/models/Password.php:190
Stack trace:
#0 /var/www/virtual/customer/html/protected/modules/massuserimport/models/UserImportContainer.php(140): humhub\modules\user\models\Password->setPassword('REDACTED')
#1 /var/www/virtual/customer/html/protected/modules/massuserimport/controllers/ImportController.php(65): humhub\modules\massuserimport\models\UserImportContainer->save()
#2 [internal function]: humhub\modules\massuserimport\controllers\ImportController->actionIndex()
#3 /var/www/virtual/customer/html/protected/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#4 /var/www/virtual/customer/html/protected/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#5 /var/www/virtual/customer/html/protected/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('index', Array)
#6 /var/www/virtual/customer/html/protected/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('massuserimport/...', Array)
#7 /var/www/virtual/customer/html/protected/vendor/yiisoft/yii2/base/Application.php(384): yii\web\Application->handleRequest(Object(humhub\components\Request))
#8 /var/www/virtual/customer/html/index.php(28): yii\base\Application->run()
#9 {main}
```
We currently have a temporary workaround by null-checking `$this->user` in `/var/www/virtual/customer/html/protected/humhub/modules/user/models/Password.php:190`

**Comment by yurabakhtin** [comment]
@weyhmueller I think this bug was already fixed in the PR https://github.com/humhub/massuserimport/pull/8 2 weeks ago.
It will be in next release 1.1.9.

**Comment by luke-** [comment]
@yurabakhtin Thanks!

I have just released version 1.1.9. 

@weyhmueller Can you confirm this version fixes the problem?

**Comment by weyhmueller** [comment]
@luke- Thank you. I can confirm that this fixes the problem.

**Comment by atdqm** [comment]
In the changelog you write Fix #8, I think it's Fix #9 ?

![image](https://user-images.githubusercontent.com/112695767/210072476-81c0ae0f-1c6d-479d-b3f6-7bc08ee75bfe.png)


https://www.humhub.com/en/marketplace/massuserimport/#changelog

I hope #8 will be correct next ?
thanks
